control-plane/stor-port/src/types/v0/store/node.rs

```diff
@@ -285,6 +287,19 @@ impl NodeSpec {
         }
     }
 
+    pub fn add_draining_volume(&mut self, volume_id: &VolumeId) {
+        self.draining_volumes
+            .insert(volume_id.clone(), SystemTime::now());
+    }
+
+    pub fn get_draining_volumes(&self) -> HashMap<VolumeId, SystemTime> {
```

Comment:
Should we return a `&HashMap<VolumeId, SystemTime>` to avoid the clones.

Reply:
fixed

Reply:
Changed to return the reference, but I think a clone() is still needed at the spec layer that calls this function.

Reply:
Yep, it could be, but its good to return at this point.